Sources/InteroperabilityTests/InteroperabilityTestCase.swift

```diff
@@ -16,7 +16,7 @@
 import GRPCCore
 
 @available(macOS 13.0, iOS 16.0, watchOS 9.0, tvOS 16.0, *)
-public protocol InteroperabilityTest {
+public protocol InteroperabilityTest: Sendable {
```

Comment:
Now that the tests have been refactored to construct the interop test in place this isn't a requirement anymore.